### ISSUE TITLE
Yosemite Dark Menu Support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [irishuang1211/macosx-nosleep-extension#139]
```
The current black and white menu bar icons are invisible on the dark menu bar 
of Yosemite.

Can you please improve it??

Steps to Reproduce
==================
1) From the preferences sub-menu in NoSleep opt to use the black and white menu 
bar icons.
2) From the system preferences opt to use dark menu bar and dock.
3) Toggle between on and off mode of NoSleep.
4) Notice neither is visible.

What Happens
============
1) Everything works fine... I just can't see the state of NoSleep and I need to 
guess where the icon is...

What should Happen
==================
1) Everything should keep working fine with the addition of me being able to 
see where the NoSleep icon is (on the menu bar) and what is the current state 
of NoSleep.



Thanks for your patience. I know it's not a "bug", but I thought it's worth 
mentioning.

Keep up the good work. It is greatly appreciated!!
```

Original issue reported on code.google.com by `ariel.a...@gmail.com` on 17 Oct 2014 at 9:46

**Comment by GoogleCodeExporter** [comment]
```
I also love the new Dark Menu, and while NoSleep doesn't have support for it,
I created a simple workaround.

Just download the icons in the ZIP file attached.
And replace the original NoSleep icons inside the application:

1. Go to the Applications folder.
2. Right-click in the NoSleep.app and select Show Package Contents.
3. Go into the Resources folder.

There you will see two PDF files with the icons.
Just copy the new icons over them, and you are ready to go.
The ZIP file attached also has the original icons, if you want to go back to 
the Light menu bar.

```

Original comment by `goo...@volanin.com.br` on 18 Oct 2014 at 10:37

Attachments:
- [NoSleepIcons.zip](https://storage.googleapis.com/google-code-attachments/macosx-nosleep-extension/issue-139/comment-1/NoSleepIcons.zip)

**Comment by GoogleCodeExporter** [comment]
```
Awesome!

Thanks a lot!!!
```

Original comment by `ariel.a...@gmail.com` on 19 Oct 2014 at 5:37

**Comment by GoogleCodeExporter** [comment]
```
Implemented in 1.4.0
```

Original comment by `integral...@gmail.com` on 15 Nov 2014 at 7:18
- Changed state: **Fixed**
